Dockerfile.develop

```diff
@@ -0,0 +1,9 @@
+FROM freqtrade:latest
```

Comment:
i think this should be based on `freqtradeorg/freqtrade:develop` or `freqtradeorg/freqtrade:latest` to keep it consistent with `Dockerfile.technical` - which is pushed / updated with each commit to develop.

for reference:  [dockerhub.com](https://hub.docker.com/r/freqtradeorg/freqtrade/?)

Reply:
done